apis/workloads/v1/instanceset_types.go

```diff
@@ -195,11 +195,6 @@ type InstanceSetSpec struct {
 	// +optional
 	Roles []ReplicaRole `json:"roles,omitempty"`
```

Comment:
Since the role probe has been removed and the ITS cannot detect the role on its own, it should be clearly specified where the ITS will obtain the role, such as the role label of pod: kubeblocks.io/role.

Reply:
I'll clearify this in #8416